docs/howto/how_to_build_and_run_python_app.md

```diff
@@ -1,14 +1,15 @@
-# How to Build and Run(Test) your Python Application
+# How to Build, Test and Run your Python Application
 
-Everything managed by pure [Bazel](https://docs.bazel.build/versions/master/be/python.html) in Apollo 6.0.
+Everything managed by pure
+[Bazel](https://docs.bazel.build/versions/master/be/python.html) in Apollo 6.0.
 
```

Comment:
Starting from Apollo 6.0, building and testing Python applications in Apollo is done using Bazel exclusively. We use [Bazel Python rules](https://docs.bazel.build/versions/master/be/python.html) to build, run, and test Python programs. This not only frees us from hand-crafting Protobuf dependencies and managing Python related Env variables manually, but also helps with managing third party Python module dependency.